docs/source/qpu.rst

```diff
@@ -9,7 +9,9 @@ them using pyQuil.
 .. note::
 
     In order to run experiments on the QPU you will need a specially upgraded API key.  If you are
-    interested in running these experiments, then email us at support@rigetti.com
+    interested in running these experiments, then email us at support@rigetti.com.
+
+    You will also need to use the following URL endpoint: https://job.rigetti.com/beta
```

Comment:
We change this endpoint when loading anything from pyquil/qpu.py.  Eventually all jobs will be routed through the same service so this information will not be required.